### PR TITLE
docs: added api key setup to README

### PR DESCRIPTION
**Pull request by vbchomp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,23 @@ npm run dev
...
 
+## Optional for using in Production Environment
+
+Request API Key from [Opensea](https://docs.opensea.io/reference/request-an-api-key)
+
+API is free to use on lmited basis. To use the API in a production environment, request key from URL above. Enter email address, project name, project description, and a link to the project website. Approval takes 1-2 days.
+
+Use API key in the header of request. Example:
+
+```Javascript
```
Comment (MuckT) [inline README.md]: Change to use Axios instead

Comment (MuckT) [reply README.md]: Check the axios calls in the public routes.

**Review by MuckT — DISMISSED** [review]
Thanks for fixing the axios example!

**Review by fadnesscharlie — APPROVED** [review]

**Review by jeremywbrazell — APPROVED** [review]

**Comment by jeremywbrazell** [comment]
merged

**Pull request merged**